docs/conf.py

```diff
@@ -49,9 +49,12 @@
 # intersphinx mapping
 intersphinx_mapping = {
     'numpy': ('https://numpy.org/doc/stable/', None),
+    'numpydoc': ('https://numpydoc.readthedocs.io/en/stable/', None),
     'scipy': ('https://docs.scipy.org/doc/scipy/', None),
     'matplotlib': ('https://matplotlib.org/stable/', None),
-    'spharpy': ('https://spharpy.readthedocs.io/en/stable/', None)
+    'spharpy': ('https://spharpy.readthedocs.io/en/stable/', None),
```

Comment:
we can remove everything instead of ``numpydoc``, ``pytest``, and ``coverage``. I've just tested it. Keeping the rest would slow down the build process(, at least a little bit).